scripts/runPostCrab.py

```diff
@@ -174,6 +212,39 @@ def main():
 #    print "report=", report
     print "done"
 
+    print("")
+
+    print "##### Get information from the output files"
+    files = []
+    for (i, lfn) in enumerate(output_files['lfn']):
+        pfn = output_files_xrootd['xrootd'][i]
+        files.append({'lfn': lfn, 'pfn': pfn})
+
+    p = os.path.dirname(output_files['lfn'][0])
+    localpath = ingridStoragePrefix + p
+
+    db_files = []
+    dataset_sumw = 0
+    dataset_nselected = 0
+    for f in files:
+        (sumw, entries) = get_file_data(f['pfn'])
+        if sumw == 0:
+            print("Warning: failed to retrieve sum of event weight for %r" % f['lfn'])
+
+        dataset_sumw += sumw
+
+        if entries == 0:
+            print("Warning: failed to retrieve number of entries for %r" % f['lfn'])
```

Comment:
So let change the DB field to nullable, and store NULL if it fails to get the number of entries instead of 0